src/modules/10-common/components/MultiTypeExecutionCondition/MultiTypeExecutionCondition.tsx

```diff
@@ -39,11 +39,14 @@ function MultiTypeMonacoTextFieldFixedTypeComponent(props: {
   )
 }
 
-function MultiTypeExecutionConditionInternal(props: MultiTypeExecutionConditionProps) {
+function MultiTypeExecutionConditionInternal(props: MultiTypeExecutionConditionProps): React.ReactElement {
   const { path, formik, allowableTypes, isInputDisabled, readonly, multiType, setMultiType, expressions } = props
 
   const conditionValue = get(formik?.values, path)
-  const allowableTypesExceptExpression = filter(allowableTypes, type => type !== MultiTypeInputType.EXPRESSION)
+  const allowableTypesExceptExpression = filter(
+    allowableTypes,
+    type => type !== MultiTypeInputType.EXPRESSION
+  ) as AllowedTypes
```

Comment:
Any reason to not just use JS's `filter`?
```suggestion
  const allowableTypesExceptExpression = allowableTypes.filter(
    type => type !== MultiTypeInputType.EXPRESSION
  )
```

Reply:
Not sure. This is existing code. I just updated the types.